exercises/sum-of-multiples/example.lua

```diff
@@ -0,0 +1,24 @@
+return function(numbers)
+  numbers = numbers or { 3, 5 }
+
+  return {
+    to = function(limit)
```

Comment:
If someone has a hard time we can move it further down, it is already at 20 or so.